### ISSUE TITLE
'str' object has no attribute _tasks

### ISSUE THREAD

**Issue by naftulikay** [prkumar/sphinx-reload#3]
I have a pretty simple Python setup in a Vagrant VM using Python 3.6.8:

```
[vagrant@ansible-role-degoss vagrant]$ which pip python
~/.pyenv/shims/pip
~/.pyenv/shims/python
[vagrant@ansible-role-degoss vagrant]$ pip --version
pip 18.1 from /home/vagrant/.pyenv/versions/3.6.8/lib/python3.6/site-packages/pip (python 3.6)
[vagrant@ansible-role-degoss vagrant]$ python --version
Python 3.6.8
[vagrant@ansible-role-degoss vagrant]$ sphinx-reload --version
v0.1.1
[vagrant@ansible-role-degoss vagrant]$ uname -a
Linux ansible-role-degoss 3.10.0-693.21.1.el7.x86_64 #1 SMP Wed Mar 7 19:03:37 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

`requirements.txt`:

```
ansible
mock
sphinx
sphinx-reload
sphinx_rtd_theme
```

When I attempt to run `sphinx-reload docs/`, I get the following exception from Python:

```
[vagrant@ansible-role-degoss vagrant]$ sphinx-reload docs/
Traceback (most recent call last):
  File "/home/vagrant/.local/bin/sphinx-reload", line 11, in <module>
    sys.exit(main())
  File "/home/vagrant/.local/lib/python3.6/site-packages/sphinx_reload.py", line 160, in main
    port=namespace.port,
  File "/home/vagrant/.local/lib/python3.6/site-packages/sphinx_reload.py", line 106, in run
    self._run(build_func, html_dir, port=port)
  File "/home/vagrant/.local/lib/python3.6/site-packages/sphinx_reload.py", line 81, in _run
    server.watch(pattern, build_func)
  File "/home/vagrant/.local/lib/python3.6/site-packages/livereload/server.py", line 211, in watch
    self.watcher.watch(filepath, func, delay, ignore=ignore)
  File "/home/vagrant/.local/lib/python3.6/site-packages/livereload/watcher.py", line 67, in watch
    'ignore': ignore,
AttributeError: 'str' object has no attribute '_tasks'
```

Is there a workaround for this?

**Comment by prkumar** [comment]
@naftulikay - I've opened #4 to fix this. This issue affects sphinx-reload with Python 3.5+, and there's no workaround as far as I can tell. After I merge #4 into `master`, I'll create a release and deploy it to `PyPI` so you can use the fix ASAP.

**Comment by prkumar** [comment]
@naftulikay - The fix should be live as part of `v0.1.2`. Please upgrade when you get the chance. Thanks!
